### DISCUSSION TITLE
columns on the page

### DISCUSSION THREAD

**Discussion by undertherain** [nunocoracao/blowfish#1051] [Q&A]
Hi!
I'm trying to make a little bit of layout on the page - 
say for the location page, to have address on the right, and map on the left side etc. The column styles from tailwind manual which work outside of the theme, somehow don't render properly when being hugo-blowfish-ed :-\
Is there any recommended (and easy) way to do something like this?

**Comment by knowinginner** [comment]
## [answer grids and components in tailwind](https://github.com/nunocoracao/blowfish/discussions/1107)

Please close the question if you find that this answer resolved the issue.

Take care

Faust

**Reply by leon1995** [reply]
I tried to

```html
<div class="grid grid-cols-2 gap-4">
  <div>
    column test 1
  </div>
  <div>
    column test 2
  </div>
</div>
```
but it doesnt work
